extensions/sns/src/commands/prepare_canisters/remove_nns_root.rs

```diff
@@ -0,0 +1,26 @@
+//! Code for executing `dfx sns prepare-canisters remove-nns-root`
+use candid::Principal;
+use clap::Parser;
+use dfx_extensions_utils::call_extension_bundled_binary;
+use std::ffi::OsString;
+use std::path::Path;
+
+/// `dfx sns prepare-canisters remove-nns-root` command line arguments.
+#[derive(Parser)]
+pub struct RemoveNnsRootOpts {
+    // TODO support multiple canisters?
+    #[arg(long)]
+    canister_id: Principal,
+}
+
+/// Executes `dfx sns prepare-canisters remove-nns-root`
+pub fn exec(opts: RemoveNnsRootOpts, dfx_cache_path: &Path) -> anyhow::Result<()> {
+    let args = vec![
+        OsString::from("prepare-canisters"),
+        OsString::from("remove-nns-root"),
+        OsString::from(opts.canister_id.to_string()),
+    ];
+
+    call_extension_bundled_binary("sns-cli", &args, dfx_cache_path)?;
```

Comment:
Does this sns-cli command have any output of interest?  `call_extension_bundled_binary` returns the command's output, but doesn't display it (a surprising choice, but that's how it is)

Reply:
Thanks for pointing this out. In both cases the binary provides success and error cases. I've added a println for both cases.

Reply:
done.